content/rs/administering/import-export/database-backup.md

```diff
@@ -34,13 +37,232 @@ Redis Software copies a backup file for each shard to the specified backup locat
+
+When saved to a local mount point or a cloud provider, backup locations need to be available to [the group and user]({{< relref "/rs/installing-upgrading/customize-user-and-group.md" >}}) running Redis Enterprise Software, `redislabs:redislabs` by default.  
+
+Redis Enterprise Software needs the ability to view permissions and update objects in the storage location. Implementation details vary according to the provider and your configuration. To learn more, consult the provider's documentation.
+
+The following sections provide general guidelines.  Because provider features change frequently, use your provider's documentation for the latest info.
+
+### FTP server
+
+Before enabling backups to an FTP server, verify that:
+
+- Your Redis Enterprise cluster can connect and authenticate to the FTP server.
+- The user specified in the FTP server location has read and write privileges.
+
+To store your backups on an FTP server, set its **Backup Path** using the following syntax:
+
+`ftp://[username]:[password]@[host]:[port]/[path]/`
+
+Where:
+
+- *protocol*: the server's protocol, can be either `ftp` or `ftps`.
+- *username*: your username, if needed.
+- *password*: your password, if needed.
+- *hostname*: the hostname or IP address of the server.
+- *port*: the port number of the server, if needed.
+- *path*: the backup path, if needed.
+
+Example: `ftp://username:password@10.1.1.1/home/backups/`
+
+The user account needs permission to write files to the server.
+
+### SFTP server
+
+Before enabling backups to an SFTP server, make sure that:
+
+- Your Redis Enterprise cluster can connect and authenticate to the SFTP server.
+- The user specified in the SFTP server location has read and write privileges.
+- The SSH private keys are specified correctly.  You can use the key generated by the cluster or specify a custom key.
+
+    When using the cluster auto generated key, copy the **Cluster SSH Public Key** to the appropriate location on the SFTP server.  This is available from the **General** tab of the **Settings** menu in the admin console.
+
+    Use the server documentation to determine the appropriate location for the SSH Public Key.
+
+To backup to an SFTP server, enter the SFTP server location in the format:
+
+```sh
+sftp://user:password@host:<:custom_port>/path/
+```
+
+For example: `sftp://username:password@10.1.1.1/home/backups/`
+
+### Local mount point
+
+Before you choose to backup to a local mount point, make sure that:
+
+- The node has network connectivity to the destination server of the mount point.
+- The `redislabs:redislabs` user has read and write privileges on the local mount point
+and on the destination server.
+- The backup location has enough disk space for your backup files. The backup files
+are saved with filenames that include the timestamp so that backup files are not overwritten.
+
+To backup to a local mount point for a node:
```

Comment:
```suggestion
To back up to a local mount point for a node:
```

Reply:
Fixed (and cleaned up a bit of nearby language).